Feature/uppsf 2376 remove annotations v1 from config

ivanruski opened the pull request:
# Description

## What

I've removed from `annotations-config.json` entries related to v1-annotations as they are no longer needed and I've updated the tests which were depending on those entries.

Also since "predicate" field must be present in any `thing` object in a given request body, it is best to include it in the
`examplePutBody.json`, otherwise sending a request using the contents of `examplePutBody.json` will result in an error.
After this change `examplePutBodyWithPredicate.json` becomes unnecessary which is why it's being removed.

## Why

[JIRA](https://financialtimes.atlassian.net/browse/UPPSF-2376)

## Scope and particulars of this PR (Please tick all that apply)

- [ ] Tech hygiene (dependency updating & other tech debt)
- [ ] Bug fix
- [x] Feature
- [ ] Documentation
- [ ] Breaking change
- [ ] Minor change (e.g. fixing a typo, adding config)

___
This Pull Request follows the rules described in our [Pull Requests Guide](https://github.com/Financial-Times/upp-docs/tree/master/guides/pr-guide)

MiroslavGatsanoga commented on annotations/model_test.go:
```diff
@@ -17,13 +17,3 @@ func TestUnMarshallingAnnotation(t *testing.T) {
 	assert.NoError(err, "Unexpected error")
 }
-
-func TestUnMarshallingAnnotationWithPredicate(t *testing.T) {
```
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏Why not leave the test to show that we need a predicate? If we don't pass predicate should the test fail or pass?

ivanruski replied:
Because this tests are about whether or not a given request body could be `Unmarshal`-ed to `Annotations` type.
There is a separate test in `cypher_test.go` which tests the case when there isn't a predicate in the request.

Also `Thing.ID` is also a mandatory field so if we decide to test for empty `predicate` here, we should test and the case for empty `ID` as well (IMO)

ivanruski replied:
> There are still a few places where it will be better to update mentions of `v1` annotations e.g. README, runbook, cypher_integration_test.go.

I've addressed this comment in a new temporary commit. Also I've found out an unused variable in `cypher.go` and decided to remove it.

MiroslavGatsanoga reviewed (commented):
There are still a few places where it will be better to update mentions of `v1` annotations e.g. README, runbook, cypher_integration_test.go.

ivanruski commented:
> There are still a few places where it will be better to update mentions of `v1` annotations e.g. README, runbook, cypher_integration_test.go.

I've addressed this comment in a new temporary commit and also I've found out an unused variable in `cypher.go` which I've removed (If this is not the place to do this, I will revert it)

ivanruski commented:
Squashing commits
